projects/server/src/main/resources/import.sql

```diff
@@ -0,0 +1,28 @@
+DELETE FROM ingredient;
```

Comment:
i think that this will not be needed

Reply:
You probably have right. I will check it.